jovian/jovian-jupyter-lab-extension/README.md

```diff
@@ -0,0 +1,166 @@
+- [Reproduce Notebooks from Jovian](https://jovian-py.readthedocs.io/en/latest/user-guide/03-reproduce.html)
+- [Notebooks as version, view Diffs between versions](https://jovian-py.readthedocs.io/en/latest/user-guide/04-version.html)
+- [Attaching utility files and model outputs with the Notebook](https://jovian-py.readthedocs.io/en/latest/user-guide/05-attach.html)
+- [Tracking Datasets, Hyperparameters and Metrics](https://jovian-py.readthedocs.io/en/latest/user-guide/06-track.html)
+- [Comparing and Analyzing all the experiments](https://jovian-py.readthedocs.io/en/latest/user-guide/07-compare.html)
+- [Collaborate with teammates/colleagues](https://jovian-py.readthedocs.io/en/latest/user-guide/08-collaborate.html)
+- [Stay connected with your model training, live updates with Slack Notifications](https://jovian-py.readthedocs.io/en/latest/jvn/notif.html)
+
+[![Documentation Status](https://readthedocs.org/projects/jovian-py/badge/?version=latest)](https://jovian-py.readthedocs.io/en/latest/?badge=latest)
+
+## Installation
+
+```
+pip install jovian --upgrade
+```
+
+> **Caution**:
+>
+> If you get a `Permission denied` error, try installing with sudo permission (on Linux/Mac).
+>
+> ```
+> sudo pip install jovian --upgrade
+> ```
+>
+> Another alternative is to try installing with the `--user` flag, but you’ll need to ensure that the target directory is added to your system `PATH`.
+>
+> ```
+> pip install jovian --upgrade --user
+> ```
+
+Visit [Docs](https://jovian-py.readthedocs.io) for more.
+
+## Contact
+
+Mail : hello@jovian.ml
+<br>
+Twitter : [@JovianML](https://twitter.com/JovianML)
+<br>
+Slack : [Slack Invite](https://bit.ly/jovian-users)
+
+## Requirements
+
+- JupyterLab >= 0.30.0
+- Jovian >= 0.1.89
+- nodejs >= 10.20.0
+
+# Jupyter Lab Extension
+
+Now you can commit your Jupyter Notebook to [Jovian](https://jovian.ml?utm_source=docs) with just **One Click**.
+Make sure you’ve completed the [Installation](../user-guide/01-install.md) before reading further.
+
+## Using Jovian Jupyter Lab Extension
+
+<img src="https://i.imgur.com/uezsdYX.png" class="screenshot" alt="jupyter extension toolbar button">
+
+Once you have successfully installed jovian, a new button `Commit` will appear on the tool bar. When using `Commit` button for first time you'll be asked to provide an API key.
+
+<img src="https://i.imgur.com/jTvA0De.png" class="screenshot" alt="jovian api key prompt">
+
+You can get the API key at [Jovian](https://jovian.ml?utm_source=docs). Once you log in, just click on `API key` button, and the key will be copied to the clipboard.
+
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button">
+
+#### Valid API key
+
+If the key is valid you will be notified with the following alert.
+
+<img src="https://i.imgur.com/lNKQO3G.png" class="screenshot" alt="valid api key prompt">
+
+#### Error with API key
+
+If the entered API key is invalid you will get following error.
+
```

Comment:
I have added minimal information here once we have updated GIFs, textual content, neat Contributing guide. We can add them here.